pipeline/train/instruction_following.py

```diff
@@ -123,7 +125,11 @@ def train_one_epoch(args, model, epoch, mimicit_loaders, tokenizer, optimizer, l
                 #     attention_mask=attention_mask.to(device_id),
                 #     max_length=256,
                 # )
-            accelerator.backward(loss_mimicit)
+            if accelerator.mixed_precision != "bf16":
```

Comment:
From my understanding. This should be judged by if we use `mix_precision`, not actually by if `mix_precision=bf16`. Since the loss is calculated within a `autocast()`.